core/editor/editor.php

```diff
@@ -788,6 +787,7 @@ public function set_post_id( $post_id ) {
 	private function get_loader() {
 		if ( ! $this->loader ) {
 			$this->loader = new Editor_Loader( Config_Provider_Factory::create() );
+			$this->loader->register_actions();
```

Comment:
change `register_actions()` to a more generic name, e.g. `register_hooks()`

Reply:
Done